lib/caps/RKL/d3d11

```diff
@@ -0,0 +1,89 @@
+###
+### Copyright (C) 2020 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+###
+### kate: syntax python;
+###
+
+# https://github.com/intel/media-driver/blob/master/docs/media_features.md
```

Comment:
This URL is for iHD driver features.  Thus, it does not apply to dxva2.  Please remove.

Reply:
Is there a documented feature list for d3d11 on Intel (RKL) somewhere that confirms the features in this caps file?  Same question for dxva2.